test/extended/apiserverauth/apiserver_util.go

```
@@ -0,0 +1,2880 @@
+	transport := &http.Transport{
+		Proxy: http.ProxyURL(proxyURL),
+		TLSClientConfig: &tls.Config{
+			RootCAs: caCertPool,
+		},
+	}
+
+	client := &http.Client{
+		Transport: transport,
+	}
+
+	url := fmt.Sprintf("https://%s:%s/healthz", fqdnName, port)
```

golangci-lint (2.12.2)

[error] host:port in url should be constructed with net.JoinHostPort and not directly with fmt.Sprintf

(nosprintfhostport)